pkgs/applications/networking/mailreaders/mblaze/default.nix

```diff
@@ -26,6 +28,7 @@ stdenv.mkDerivation rec {
     install -Dm644 -t $out/share/zsh/site-functions contrib/_mblaze
   '' + lib.optionalString (ruby != null) ''
     install -Dt $out/bin contrib/msuck contrib/mblow
+    wrapProgram $out/bin/mcom --prefix PATH : ${lib.makeBinPath [ file ]}
```

Comment:
For purity, we should also wrap it with other stuff, like awk, even if it’s likely to be in PATH. Maybe have a look at the source code and see what it needs?

You should be able to test with

```sh
EDITOR=$(command -v nano) PATH= $(command -v mcom)
```

Reply:
Thanks for the heads-up about my email adress, I registered it to github.

I was just thinking about what you're saying in your second comment. I put a [question](https://discourse.nixos.org/t/how-to-test-the-result-of-a-build-in-a-pure-environment/8497) on discourse.nixos.org about that. Do you know of a way to run an executable that results from a nix-build in a "pure" environment, where only the executable dependencies are present and none of the executables that I have installed in my user space?

Reply:
I just realized that the piece of code is the answer to my last question…

Reply:
> I just realized that your piece of code is the answer to my last question…